dashboard/config/initializers/backtrace_silencers.rb

```diff
@@ -5,3 +5,19 @@
 
 # You can also remove all the silencers if you're trying to debug a problem that might stem from framework code.
 # Rails.backtrace_cleaner.remove_silencers!
+
+# silence annoying deprecations
+silenced = [
+  /ActionController::TestCase HTTP request methods/,
+  /ActionDispatch::IntegrationTest HTTP request methods/,
+  /`render :text` is deprecated/,
+  /alias_method_chain is deprecated/
+]
```

Comment:
These can be done after the Rails 5 upgrade is merged but at a lower priority. Most of them need to be done before the next minor-version upgrade (Rails 5.1), I think. The reason they aren't included in the current PR is because they would require refactoring/rewriting practically our entire Rails test suite to match the new syntax, which would have bloated this PR too much (quite large and hard to keep updated long enough to merge as is).

I may have already finished the `render :text` deprecations and can remove that line.